src/main/java/uk/gov/hmcts/reform/idam/web/config/MessagesConfiguration.java

```diff
@@ -1,30 +1,27 @@
 package uk.gov.hmcts.reform.idam.web.config;
 
-import java.util.Locale;
-
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.LocaleResolver;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
 import org.springframework.web.servlet.i18n.LocaleChangeInterceptor;
-import org.springframework.web.servlet.i18n.SessionLocaleResolver;
 
 @Configuration
 public class MessagesConfiguration implements WebMvcConfigurer {
 
     @Bean
     public LocaleResolver localeResolver() {
-        SessionLocaleResolver slr = new SessionLocaleResolver();
-        slr.setDefaultLocale(Locale.UK);
-        return slr;
+        return new CookieLocaleResolver();
     }
 
     @Bean
     public LocaleChangeInterceptor localeChangeInterceptor() {
-        LocaleChangeInterceptor lci = new LocaleChangeInterceptor();
-        lci.setParamName("lang");
-        return lci;
+        final LocaleChangeInterceptor interceptor = new LocaleChangeInterceptor();
+        interceptor.setParamName("locale");
```

Comment:
Can we use this?
https://openid.net/specs/openid-connect-core-1_0.html
![image](https://user-images.githubusercontent.com/47384516/76746899-24d29500-6770-11ea-85e3-de935900722b.png)

Reply:
Yes, with some code changes.

Reply:
I guess what I am saying is, lets use this 😁

Reply:
Done.